<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>579</width>
                <height>418</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>30</y>
                        <width>411</width>
                        <height>291</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="main_vbox_layout">
                    <item>
                        <widget class="QPushButton" name="sign_in_button">
                            <property name="text">
                                <string>登录</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="exit_button">
                            <property name="text">
                                <string>退出</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>579</width>
                    <height>24</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
